src/core/link.cpp

```diff
@@ -999,18 +999,28 @@ namespace mamba
         }
 
         reproc::options options;
+        reproc::redirect silencer;
+        silencer.type = reproc::redirect::pipe;
+        options.redirect.out = silencer;
+        options.redirect.err = silencer;
+        std::string out, err;
+
         options.redirect.parent = true;
         std::string cwd = m_context->target_prefix;
         options.working_directory = cwd.c_str();
 
         auto [wrapped_command, script_file]
             = prepare_wrapped_call(m_context->target_prefix, command);
+
         LOG_INFO << "Running wrapped python compilation command " << join(" ", command);
-        auto [_, ec] = reproc::run(wrapped_command, options);
+        auto [_, ec] = reproc::run(
+            wrapped_command, options, reproc::sink::string(out), reproc::sink::string(err));
 
-        if (ec)
+        if (ec || !err.empty())
         {
-            throw std::runtime_error(ec.message());
+            LOG_INFO << "noarch pyc compilation failed (cross-compiling?). " << ec.message();
+            LOG_INFO << err;
+            return {};
```

Comment:
I would prefer to warn on error that are authorized to fail, it's not a regular info log level

Reply:
it would clutter the logs for cross-compilation a lot. Also this is actually mostly harmless.